src/translations/en.json

```diff
@@ -925,6 +925,9 @@
     "DefaultOutageRate": "Default outage rate",
     "GeneratorsWithoutOutage": "Generators without outage simulation",
     "GeneratorsWithFixedActivePower": "Generators with fixed active power",
+    "GeneratorsFilters": "GENERATOR FILTER",
```

Comment:
should change "GeneratorsFilters" to "GeneratorFilter" ? (remove the "s")

Reply:
Renamed